examples/example13-timezone.html

```diff
@@ -37,25 +37,29 @@
   <script src="https://uicdn.toast.com/tui.time-picker/v2.0.3/tui-time-picker.min.js"></script>
   <script src="https://uicdn.toast.com/tui.date-picker/v4.0.3/tui-date-picker.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.31/moment-timezone-with-data.js" integrity="sha512-ECoTMVFwwVtxjEBRjUMjviUd6hBjwDhBJI0+3W2YDs+ld5rHHUDr59T15gxwEPkGu5XLmkASUSvPgQe/Tpyodg==" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/chance/1.0.13/chance.min.js"></script>
   <script src="../dist/tui-calendar.js"></script>
   <script src="./js/data/calendars.js"></script>
   <script src="./js/data/schedules.js"></script>
   <script type="text/javascript" class="code-js">
   var cal = new tui.Calendar('#calendar', {
     defaultView: 'week', // set 'week' or 'day'
-    timezones: [         // set timezone config
+    timezoneOffsetFn: function(timezone, timestamp) {
+      return -moment.tz.zone(timezone).utcOffset(timestamp); // e.g. +09:00 => 540, -04:00 => -240
+    },
+    timezones: [ // set timezone config
       {
-        timezoneOffset: 540,
-        tooltip: 'Seoul'
+        tooltip: 'New York',
+        timezone: 'America/New_York',
       },
       {
-        timezoneOffset: -420,
-        tooltip: 'Los Angeles'
+        timezoneOffset: 540,
+        tooltip: 'Seoul',
       }
     ],
     taskView: false,
-    scheduleView: ['time']
+    scheduleView: ['time'],
```

Comment:
trailing comma can't be used in case of old Internet Explorer. This example will be run on old Internet Explorer.